shlr/sdb/src/sdb.c

```diff
@@ -184,7 +184,7 @@ SDB_API int sdb_count(Sdb *s) {
 	return count;
 }
 
-static void sdb_fini(Sdb* s, int donull) {
+static void sdb_fini(Sdb* s, bool donull) {
```

Comment:
those files are from a different repository. dont patch them in here. make the PR in https://github.com/radareorg/sdb